text/chapter13.md

```diff
@@ -332,7 +329,7 @@ We have to write a function which perturbs a random generator given a value of t
   coarbitrary Leaf = id
 ```
 
-If the tree is a `Branch`, then we will perturb the generator using the left subtree, the value and the right subtree, using function composition to create our perturbing function:
+If the tree is a `Branch`, then we will perturb the generator using the left subtree, the value, and the right subtree. Function composition creates our perturbing function:
```

Comment:
This construction seems a bit odd to me, I think the previous text is clearer. The creator of the perturbing function function is ‘us,’ surely?

Reply:
Yes. I was getting confused by lack of the oxford comma when reading it. Then it seemed like the sentence should be broken-up.

What about changing the second line to:
> We use function composition to create our perturbing function: